packages/dds/tree/src/test/shared-tree-core/sharedTreeCore.spec.ts

```diff
@@ -0,0 +1,185 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { IEvent } from "@fluidframework/common-definitions";
+import { TypedEventEmitter } from "@fluidframework/common-utils";
+import {
+    IChannelAttributes,
+    IChannelStorageService,
+} from "@fluidframework/datastore-definitions";
+import { ISummaryTree, SummaryObject, SummaryType } from "@fluidframework/protocol-definitions";
+import {
+    ITelemetryContext,
+    ISummaryTreeWithStats,
+    IGarbageCollectionData,
+} from "@fluidframework/runtime-definitions";
+import { mergeStats } from "@fluidframework/runtime-utils";
+import { MockFluidDataStoreRuntime, MockSharedObjectServices } from "@fluidframework/test-runtime-utils";
+import {
+    Index,
+    SharedTreeCore,
+    SummaryElement,
+    SummaryElementParser,
+    SummaryElementStringifier,
+} from "../../shared-tree-core";
+import { AnchorSet } from "../../tree";
+import { DefaultChangeSet, DefaultRebaser } from "../../feature-libraries";
+
+describe("SharedTreeCore", () => {
+    it("summarizes without indexes", async () => {
+        const tree = createTree();
+        const { summary, stats } = await tree.summarize();
+        assert(summary);
+        assert(stats);
+        assert.equal(stats.treeNodeCount, 1);
+        assert.equal(stats.blobNodeCount, 0);
+        assert.equal(stats.handleNodeCount, 0);
```

Comment:
if this covers all the stats, you can do one:

```suggestion
        assert.deepEqual(stats, { treeNodeCount: 1, blobNodeCount: 0, handleNodeCount: 0 })
```

Reply:
There are two other properties (blob size) that I didn't want to test.